eng/NuGetPackageVerifier.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Delete this file. There is already a NuGetPackageVerifier.json file in the repo root.